geth/node/rpc_test.go

```diff
@@ -32,10 +47,69 @@ func (s *RPCTestSuite) SetupTest() {
 
 	nodeManager := node.NewNodeManager()
 	require.NotNil(nodeManager)
-
 	s.NodeManager = nodeManager
 }
 
+func (s *RPCTestSuite) TestRPCSendTransaction() {
+	require := s.Require()
+	expectedResponse := []byte(`{"jsonrpc": "2.0", "status":200, "result": "3434=done"}`)
+
+	// httpRPCServer will serve as an upstream server accepting transactions.
+	httpRPCServer := httptest.NewServer(service{
+		Handler: func(w http.ResponseWriter, r *http.Request) {
+			defer r.Body.Close()
+
+			var txReq txRequest
+
+			err := json.NewDecoder(r.Body).Decode(&txReq)
+			require.NoError(err)
+
+			switch txReq.Method {
+			case "eth_getTransactionCount":
```

Comment:
Are there any other possible `txReq.Method` values in this test that we care about? If not maybe use `if`?